beacon-chain/blockchain/service.go

```diff
@@ -58,6 +58,7 @@ type Service struct {
 	headLock               sync.RWMutex
 	stateNotifier          statefeed.Notifier
 	genesisRoot            [32]byte
```

Comment:
What is the differnce between this and the one below?

Reply:
thats the genesis state root, which is different from the genesis validator root.